src/style.css

```
@@ -0,0 +1,145 @@
+body {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  padding-bottom: 16px;
+  padding-top: 32px;
```

You add here all paddings and delete paddings form card__description

```suggestion
  padding: 32px 16px 16px;
```